test_eos-phone-home.py

```diff
@@ -0,0 +1,159 @@
+import imp
+import os
+
+import pytest
+
+HERE = os.path.dirname(__file__)
+eos_phone_home_path = os.path.join(HERE, 'eos-phone-home')
+eos_phone_home = imp.load_source("m", eos_phone_home_path)
+
+
+@pytest.fixture
+def phone_home(tmpdir):
+    return eos_phone_home.PhoneHome(True, False, root=tmpdir.strpath)
+
+
+def test_instantiate(tmpdir, phone_home):
+    assert tmpdir.strpath == phone_home._root
+
+
+def test_get_product_info_arm(tmpdir, phone_home):
+    compatible = tmpdir.join('proc', 'device-tree', 'compatible')
+    compatible.write_binary(b'vendor1,model1\0vendor2,model2\0',
+                            ensure=True)
+
+    ssn = tmpdir.join('sys', 'class', 'endless_mfgdata', 'entries', 'SSN')
+    ssn.write('  serial  ', ensure=True)
```

Comment:
Nit: `ssn` is a commonly used acronym for Social Sec Num in the US so maybe would be good to change it to s/*/serial, serial_num, or something else.

Reply:
I don't really agree because the file in `/sys` is called `SSN` and we can't change that. But, okay.

Reply:
`endless_serial_number_file`